scripts/install_composable.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+echo "Composable Multiplatform installer"
+myos=$(uname -a)
+echo $myos
+
+# Ubuntu:
+if echo "$myos" | fgrep -q Ubuntu 2>/dev/null ; then      
```

Comment:
not sure. is deploy to parachain nodes happens via docker or via nix?  would each developer manually handle deps?

Reply:
well yeah, there is a docker compose file and nix build, The goal of this script is that a new user is suppose to be able to just spin up a virtual machine(or rent a vps) and be able to install the node with a one liner, so the finished product will look like: curl -sSH get.composable.finance | sh, then that will kick in this script

Reply:
hm, so instead of installing nix/docker and than run from that, he runs from script. 
additionally we have polkadot launcher to run on dev.

to be able to be our parachain, you have to be pemissioned afaik, so user will not be able just plug his stuff into our consensus? (until it is collator? collators - the more the better).

would you be able to describe target audience of this script?

i will try to run it on my wsl2(sorry, i am windows person) if that works for me later:)

Reply:
@dzmitry-lahoda, sure thing, so the inspiration comes from how substrate has a oneliner to install and build it(curl -sSH https://getsubstrate.io/ | sh), would be cool to have a one liner a user can use to install and compile the node in order to make it easy for future node operators